test/unit/utIOStreamBuffer.cpp

```diff
@@ -86,7 +84,7 @@ TEST_F( IOStreamBufferTest, open_close_Test ) {
     const auto dataCount = dataSize / sizeof(*data);
 
     char fname[]={ "octest.XXXXXX" };
-    auto* fs = MakeTmpFile(fname);
+    FILE *fsm = fopen(fname, "w+");
```

Comment:
This is not even remotely correct. `MakeTmpFile` securely creates a file which did not exist previously.

Reply:
Right!It was a test to see if this will influence the failing unittest somehow. Any ideas what is going wrong there?

Reply:
The `fopen` thing? That's MS braindeadedness, just add `_CRT_SECURE_NO_WARNINGS` preprocessor macro somewhere in CMakelists.txt

Reply:
No, the failed unittest with the error code 127 when opening the tempfile.

Reply:
Looks like it assumes that MSVC is the same as win32 and GCC/clang isn't.

https://github.com/assimp/assimp/blob/f63625256c4e6f18fca8e7dc857b47a02320e867/test/unit/UnitTestFileGenerator.h#L53